static/templates/confirm_dialog/confirm_deactivate_user.hbs

```diff
@@ -1,7 +1,29 @@
 <p>
     {{#tr}}
         When you deactivate <z-user></z-user>, they will be immediately logged out.
-        {{#*inline "z-user"}}<strong>{{username}}</strong>{{#if email}} &lt;{{email}}&gt;{{/if}}{{/inline}}
+        {{#*inline "z-user"}}<strong>{{username}}</strong>{{#if email}} &#40;{{email}}&#41;{{/if}}{{/inline}}
     {{/tr}}
 </p>
 <p>{{t "Their password will be cleared from our systems, and any bots they maintain will be disabled." }}</p>
+<label class="checkbox">
+    <input type="checkbox" class="send_email" data-key="{{ key }}" />
+    {{#tr}}
+    Send a notification email to <z-user></z-user>?
+    {{#*inline "z-user"}}<strong>{{username}}</strong>{{#if email}} &#40;{{email}}&#41;{{/if}}{{/inline}}
```

Comment:
I think it'd be cleaner here to just have the name, and not also the email address; we provided the email elsewhere and it's nicer for this to be short.